views/fe-utils/fetchUtil.js

```diff
@@ -88,26 +91,29 @@ const createStudyDetails = async (body) => {
 }
 
 const deleteStudyDetails = async (id) => {
-  const res = await window.fetch(`${routes.basePath}/api/v1/study-details/${id}`, {
-    ...defaultApiOptions,
-    method: 'DELETE',
-  })
+  const res = await window.fetch(
+    `${routes.basePath}/api/v1/study-details/${id}`,
```

Comment:
could this live in the routes object as something like `routes.studyDetails(id: number)`

Reply:
or maybe `routes.api.studyDetails(id: number)`

Reply:
https://github.com/AMP-SCZ/dpdash/pull/118/commits/a02b5f9495eed90745bb37c6f90df4df8e963870